examples/stress_tests/many_cubes.rs

```diff
@@ -51,14 +51,22 @@ struct Args {
     #[argh(option, default = "1")]
     mesh_count: usize,
 
-    /// whether to disable frustum culling. Stresses queuing and batching as all mesh material entities in the scene are always drawn.
+    /// whether to disable all frustum culling. Stresses queuing and batching as all mesh material entities in the scene are always drawn.
     #[argh(switch)]
     no_frustum_culling: bool,
 
     /// whether to disable automatic batching. Skips batching resulting in heavy stress on render pass draw command encoding.
     #[argh(switch)]
     no_automatic_batching: bool,
 
+    /// whether to enable GPU culling.
```

Comment:
Can we model these options as an enum with `Culling::Cpu/Gpu/None`?

Reply:
Well, it'd have to be four options: no culling, CPU culling only, GPU culling only, and both CPU and GPU culling.(The last one will matter once the GPU can do occlusion culling and the CPU can't.) So that's why I made it two bools.

Reply:
Admittedly this PR puts us in kind of a weird state, with a GPU culling feature that almost never helps, but that's because this PR is really a foundation for further improvements (that are themselves blocked on other PRs, and would be large).